files/scripts/core_cleanup.py

```diff
@@ -19,11 +35,20 @@ def main():
         logger.log_error('Root required to clean up core files')
         return
 
+    expire_date = datetime.now() - timedelta(days=EXPIRE_DAYS)
+
     logger.log_info('Cleaning up core files')
     core_files = [f for f in os.listdir(CORE_FILE_DIR) if os.path.isfile(os.path.join(CORE_FILE_DIR, f))]
 
     core_files_by_process = defaultdict(list)
     for f in core_files:
+        # delete expired core files
```

Comment:
>delete expired core files [](http://example.com/codeflow?start=10&length=25)

The expired criterial will be tricky if NTP jump. Better to keep old algorithm "keep recent MAX_CORE_FILES".

Reply:
Fixed